projects/bp-gallery/src/views/gallery/shared/add-pictures-to-collection.hook.ts

```diff
@@ -11,6 +13,9 @@ import {
  *  - set the pictures array on that collection
  */
 const useAddPicturesToCollection = () => {
```

Comment:
This hook is not used anymore, as you put an equivalent into the `manage-collection-pictures.hook.ts` file, so can we delete this file here?